arrows/tests/test_resection_camera.h

```diff
@@ -0,0 +1,151 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include <vital/math_constants.h>
+#include <arrows/mvg/projected_track_set.h>
+#include <test_scene.h>
+#include <test_eigen.h>
+
+using namespace kwiver::vital;
+using namespace kwiver::arrows::mvg;
+using namespace std;
+
+// ----------------------------------------------------------------------------
+// Test camera resection with ideal points.
+TEST(resection_camera, ideal_points)
+{
+  // landmarks at random locations
+  landmark_map_sptr landmarks = kwiver::testing::init_landmarks(128);
+  landmarks = kwiver::testing::noisy_landmarks(landmarks, 1.0);
+
+  // camera sequence (elliptical path)
+  camera_map_sptr cameras = kwiver::testing::camera_seq();
+
+  // tracks from projections
+  track_set_sptr tracks = projected_tracks(landmarks, cameras);
+
+  const frame_id_t frmID = 0;
```

Comment:
```suggestion
  constexpr frame_id_t test_frame = 0;
```
(Likewise in the other test.)